aptos-move/framework/aptos-framework/sources/timestamp.move

```diff
@@ -164,11 +164,16 @@ module aptos_framework::timestamp {
     }
 
     #[test_only]
-    public fun update_global_time_for_test(timestamp: u64) acquires CurrentTimeMicroseconds {
+    public fun update_global_time_for_test(timestamp_microsecs: u64) acquires CurrentTimeMicroseconds {
```

Comment:
This was very confusing before as most modules usually obtain timestamps in seconds, while this takes timestamp in microseconds.